cluster/config-defaults.yaml

```diff
@@ -111,7 +111,7 @@ prometheus_mem_min: "2Gi"
 prometheus_cpu_min: "0"
 prometheus_remote_write: "disabled"
 prometheus_tsdb_retention_size: "disabled"
-prometheus_csi_ebs: "true"
+prometheus_csi_ebs: "false"
```

Comment:
This seem to not be in use anymore. The places where it was used was changed to `enable_csi_migration`

Reply:
It's still used to create the prometheus stateful set.

Reply:
Ah got it! :+1: